buildtest/cli/report.py

```diff
@@ -126,6 +129,17 @@ def __init__(
         self.color = color
         self.input_report = report_file
 
+        try:
+            report=Report(configuration=configuration, format_detailed=True, format_args="name,id,user")
+        except argparse.ArgumentError:
+            console.print_exception()
+            return True
```

Comment:
you dont call the `Report` class in its own __iniit__ method thats silly :-) 

What you need to do is the following

```python

  if self.format_detailed and format_args:
     raise BuildTestError("You cannot specify --detailed and --format")
```